src/services/deleteBlacklistedToken.js

```diff
@@ -0,0 +1,25 @@
+import Sequelize from 'sequelize';
+import db from '../database/models';
+
+
+const { BlacklistedToken } = db;
+const { Op } = Sequelize;
+
+const deleteExpiredBlacklistedToken = async () => {
+  const todaysDate = new Date();
+  const currentUnixTime = Math.floor(todaysDate.getTime() / 1000);
+  const tokens = await BlacklistedToken.findAll({
+    where: { expTime: { [Op.lte]: currentUnixTime.toString() } }
+  });
+
+  const tokenArray = tokens.map(value => value.id);
+  BlacklistedToken.destroy({
+    where: {
+      id: {
+        [Op.in]: tokenArray
```

Comment:
is the `[Op.in]` really needed here? Doesn't `BlacklistedToken.destroy({ where: { id: tokenArray })` work?

Reply:
Thank you for the feedback. I didn't know I could actually do without the [Op.in]. I will work on it. Thanks